source/opt/loop_peeling.cpp

```diff
@@ -1063,7 +1063,7 @@ LoopPeelingPass::LoopPeelingInfo::HandleInequality(CmpOperator cmp_op,
   }
 
   uint32_t cast_iteration = 0;
-  // sanity check: can we fit |iteration| in a uint32_t ?
+  // coherence check: can we fit |iteration| in a uint32_t ?
```

Comment:
I'd prefer "integrity check" instead of "coherence check".

SPIR-V already has a "Coherent" decoration for a particular memory ordering constraint.

Reply:
Done.